features/step_definitions/basic_steps.rb

```diff
@@ -22,23 +22,25 @@
 
 Given('We have the following recipes:') do |table|
   table.hashes.each do |recipe|
-    if recipe[:user] && recipe[:image]
-      user = User.find_by email: recipe[:user]
-      recipe_hash = recipe.except('user', 'image')
-      new_recipe = create(:recipe, recipe_hash.merge(user: user))
+    if recipe[:user]
+      recipe[:user] = User.find_by email: recipe[:user]
+    end
+
+    if recipe["forked from"]
+      recipe[:original_recipe_id] = Recipe.find_by(title: recipe["forked from"]).id if recipe["forked from"] != ""
+      recipe.delete("forked from")
+    end
```

Comment:
I still don't understand this conditional

Reply:
In order to keep the recipes table not technical, we added the column `forked from`, as follow:
```
      | title                 | user                      | forked from  |
      | Pizza              | maran@test.com |                     |
      | Forked Pizza  | maran@test.com | Pizza           |
```
If a recipe is forked from another, like the second case, then its attribute `original_recipe_id` need to be updated.
A more readable version of the condition statement would be ass follow: 
```
if recipe["forked from"].exists?
  original_recipe_title = recipe["forked from"]

  unless original_recipe_name.empty?
    original_recipe_id = Recipe.find_by(title: original_recipe_title).id
  end

  recipe.delete("forked from")
  recipe[:original_recipe_id] = original_recipe_id
end
```

Reply:
Okay. Let's mention that in PT and create a chore that we can/should clarify some of the step definitions.